cawg_identity/src/tests/identity_assertion/validation_method/continue_when_possible.rs

```diff
@@ -39,6 +39,7 @@ use crate::{x509::X509SignatureVerifier, IdentityAssertion};
 async fn malformed_cbor() {
     let format = "image/jpeg";
     let test_image = include_bytes!("../../fixtures/validation_method/malformed_cbor.jpg");
+    const IDENTITY_URI: &str = "self#jumbf=c2pa.assertions/cawg.identity";
```

Comment:
Will this be correct/updated for identity assertions other than the active manifest?